core/trino-main/src/test/java/io/trino/execution/scheduler/TestBroadcastPipelinedOutputBufferManager.java

```diff
@@ -54,7 +54,7 @@ public void test()
 
         // try to set no more buffers again, which should not result in an error
         // and output buffers should not change
-        hashOutputBufferManager.addOutputBuffer(new OutputBufferId(6));
+        hashOutputBufferManager.noMoreBuffers();
```

Comment:
Emm, The comment before this line indicate:

> try to set no more buffers again, which should not result in an error
and output buffers should not change

Before b4f40ab9 we can only set `noMoreBuffers` via `addOutputBuffers(newBuffers, true)`, but now we only need simply call `noMoreBuffers()`. So I think it may not be necessary especially we have already tested `addOutputBuffer(new OutputBufferId(5))`. WDYT?